src/Skeletor/App/Config/SkeletorConfigurator.php

```diff
@@ -62,4 +64,13 @@ public function getConfig()
     {
         return $this->options;
     }
+
+    /**
+     * @param array $options
+     */
+    public function setConfig(array $options)
```

Comment:
should we rename this method, we are saving/storing the config, setConfig implies you would be doing something like $this->config = $options